SonarQube.Client/ISonarQubeService.cs

```diff
@@ -81,5 +81,11 @@ public interface ISonarQubeService
         Task<SonarQubeHotspot> GetHotspotAsync(string hotspotKey, CancellationToken token);
 
         Task<IList<SonarQubeIssue>> GetTaintVulnerabilitiesAsync(string projectKey, CancellationToken token);
+
+        /// <summary>
+        /// Returns the URI to view the specified issue on the server
+        /// </summary>
+        /// <remarks>The method does not check whether the project or issue exists or not</remarks>
+        Uri GetViewIssueUrl(string projectKey, string issueKey);
```

Comment:
Unfortunately this won't work for hotspots; there's a separate URL for them.
I was going to add an API for hotspots at the same time but decided to do it separately, partly because I couldn't decide how to handle requests for earlier versions of SQ i.e. whether to do any version checks or not. Perhaps there's no need; if the user is on an earlier version of SQ, there's no way in the UI they'd get to the point where we'd want to get a "view hotspot" URL.